[2022-01-25 06:35:59,707] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-01-25 06:35:59,724] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-01-25 06:35:59,725] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:35:59,725] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 06:35:59,725] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:35:59,739] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-16 00:00:00+00:00
[2022-01-25 06:35:59,747] {standard_task_runner.py:52} INFO - Started process 3886 to run task
[2022-01-25 06:35:59,764] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-16T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp4vnlsu6l', '--error-file', '/tmp/tmp3f3fzohj']
[2022-01-25 06:35:59,768] {standard_task_runner.py:77} INFO - Job 6: Subtask first_task
[2022-01-25 06:35:59,836] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-16T00:00:00+00:00 [running]> on host ceab856ace11
[2022-01-25 06:36:00,013] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-16T00:00:00+00:00
[2022-01-25 06:36:00,013] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 06:36:00,014] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 06:36:00,022] {subprocess.py:85} INFO - Output:
[2022-01-25 06:36:00,024] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 06:36:00,025] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 06:36:00,050] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220116T000000, start_date=20220125T063559, end_date=20220125T063600
[2022-01-25 06:36:00,088] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 06:36:00,121] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 07:27:42,962] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-01-25 07:27:42,977] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-01-25 07:27:42,977] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:42,977] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 07:27:42,977] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:42,990] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-16 00:00:00+00:00
[2022-01-25 07:27:42,995] {standard_task_runner.py:52} INFO - Started process 345 to run task
[2022-01-25 07:27:43,012] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-16T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpwo185hcz', '--error-file', '/tmp/tmpg20jusm5']
[2022-01-25 07:27:43,016] {standard_task_runner.py:77} INFO - Job 6: Subtask first_task
[2022-01-25 07:27:43,078] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-16T00:00:00+00:00 [running]> on host 943a4c094194
[2022-01-25 07:27:43,267] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-16T00:00:00+00:00
[2022-01-25 07:27:43,268] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 07:27:43,268] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 07:27:43,277] {subprocess.py:85} INFO - Output:
[2022-01-25 07:27:43,280] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 07:27:43,280] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 07:27:43,308] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220116T000000, start_date=20220125T072742, end_date=20220125T072743
[2022-01-25 07:27:43,333] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 07:27:43,363] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
